service/service.go

```diff
@@ -44,12 +44,20 @@ func New(conf *config.Config, oracleAccount *panacea.OracleAccount) (*Service, e
 
 	grpcClient, err := panacea.NewGrpcClient(conf)
 	if err != nil {
+		if err := queryClient.Close(); err != nil {
```

Comment:
I think there is typo. It seemed to be `grpcClient`.

Reply:
It's not typo.
If an error occurs in creating new gRPC client, we need to close the resources previously initialized.

Reply:
Ah, okay. I understand now. Thanks! 👍